data/packed/us.json

```diff
@@ -67,7 +66,6 @@
 		"America/Indiana/Knox|US/Indiana-Starke",
 		"America/Kentucky/Louisville|America/Louisville",
 		"America/Los_Angeles|US/Pacific",
-		"America/Los_Angeles|US/Pacific-New",
```

Comment:
is this intended?

Reply:
Yes, this pacific-new is not present in 2021e. It was present though in 2017b